grammars/less.cson

```diff
@@ -278,15 +306,14 @@
         'name': 'punctuation.section.property-list.end.css'
     'comment': 'Match empty braces to give proper ↩ action'
     'match': '(\\{)(\\})'
-    'name': 'meta.brace.curly.less'
+    'name': 'meta.brace.curly.css'
   }
   {
     'match': '\\{|\\}'
-    'name': 'meta.brace.curly.less'
+    'name': 'meta.brace.curly.css'
   }
   {
-    'match': '\\(|\\)'
-    'name': 'meta.brace.round.less'
+    'include': '#brace_round'
```

Comment:
This is the only one that is used more that once. I'm on the fence about this. Moving the other braces to the repository as well would be consistent, but it also adds code. If you have a clear preference I don't mind changing it.